### PR TITLE
feat: switch various components to use `@brightspace-ui/core`

### PR DESCRIPTION
**Pull request by devpow112**
All of these were simply drop in replacements as the files from the individual packages simply had a single line that was an import from `@brightspace-ui/core`.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+14
```
Comment (devpow112) [inline .nvmrc:1]: Pinning to `node 14`/`npm 6` as `npm 8` has issues with installs and git style refs. The default for runners in now `node 16`/`npm 8` if you don't specify a version.

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "type": "git",
     "url": "https://github.com/BrightspaceUI/d2l-my-dashboards-ui.git"
   },
-  "version": "3.0.6",
+  "version": "3.1.0",
```
Comment (devpow112) [inline package.json:7]: No auto release on this repo so will have to manually release.

**Comment by devpow112** [comment]
This will remove the last reference of `d2l-card` from BSI 🎉.

**Review by dlockhart — APPROVED** [review]
Wow lots of references cleaned up in this one, awesome!

**Pull request merged**